ctx.go

```diff
@@ -1978,3 +1978,8 @@ func (c *DefaultCtx) setMatched(matched bool) {
 func (c *DefaultCtx) setRoute(route *Route) {
 	c.route = route
 }
+
+func (c *DefaultCtx) Drop() error {
```

Comment:
Please add a doc comment here explaining what `Drop()` does. You can use the same description you wrote for the feature in `docs/api/ctx.md`.

Reply:
Ok i'll make the changes later